general/getting-started-in-tech.md

```diff
@@ -0,0 +1,99 @@
+    - Calculating the position of your vehicle and detecting collisions
+    - Validating that your account has enough funds to send this payment and processing the payment
+    - Starting or stopping your heating or air conditioning depending on the current temperature and the target you set
+
+At codebar, we concentrate on web technologies for several reasons:
+
+- No initial investment is needed: all the tools you need to learn about web technologies are available online for free. And the computer you use doesn’t have to be new or a top of the range model, most computers will perform well, including shared computers in libraries.
+- You can see results very fast.
+- They are ubiquitous: a lot of technologies that started on the web are now used in many other places. Even if you end up working in a domain that has nothing to do with the web, you will likely use or interact with web technologies.
+
+Web technologies are designed in such a way that they tend to more clearly separate user interface work from data processing work. As a result, web developer jobs typically fall into three broad categories:
+- Front end: work that deals with user interfaces
+- Back end: work that deals with data processing
+- Full stack: work that combines both aspects
+
+Both front-end and back-end have their own complexities and neither is easier than the other. So let’s have a look at what they both involve. Take this web page as an example. It is a page on the codebar.io website and you are reading it via a web browser on a computer or mobile device. The codebar website runs on a computer in a data center that runs a piece of software called a web server. Both talk to each other via the internet.
+
+To build a website like this involves two pieces of code:
+- The front-end code that runs in the web browser
+- The back-end code that runs on the web server
+
+Below we go into more depth about these roles, and what languages you would need to learn.
+
+## Front-end Development
+
+As a front-end developer you would be responsible for the code that runs in the web browser and displays information to visitors. It is typically the practice of writing HTML, CSS and JavaScript.
+
+### HTML (Hyper Text Markup Language)
+
+HTML is the language used to build websites. All text and content that you see on the internet is built using HTML.
+
+### CSS (Cascading Style Sheets)
+
+CSS is used alongside HTML to style the page and make it look good.
+
+
+### JavaScript
+
+JavaScript is used to change what is displayed on a web page in response to user activity.
+
+## Back-end Development
+
+As a back-end developer you would be responsible for the code that runs on the server. This type of development usually consists of three parts: a server, an application, and a database. Many back-end developers know front-end languages such as HTML and CSS but need to use languages such as Ruby on Rails, Python, PHP, Java, and .NET to get the job done.
```

Comment:
@KimberleyCook I just noticed this but I think here we should say Ruby instead of Ruby on Rails. Ruby is the language and Rails is the framework built on top of it.